packages/react/src/FieldSet/FieldSet.tsx

```diff
@@ -6,22 +6,41 @@
 import clsx from 'clsx'
 import { forwardRef } from 'react'
 import type { ForwardedRef, HTMLAttributes, PropsWithChildren } from 'react'
+import { LabelOptionalFlag } from '../Label/Label'
```

Comment:
Seeing as we currently define the styling of the legend separately from the styling of the label, maybe we shouldn't reuse the styling for the flag here? It's a bit confusing if only part of the styling is reused imo.